spec/unit/installer/xcode/pods_project_generator_spec.rb

```diff
@@ -80,44 +80,44 @@ class Xcode
 
             @installation_options = Pod::Installer::InstallationOptions.new
 
-            @generator = PodsProjectGenerator.new(config.sandbox, aggregate_targets, pod_targets, @analysis_result,
+            @single_project_generator = SinglePodsProjectGenerator.new(config.sandbox, aggregate_targets, pod_targets, @analysis_result,
```

Comment:
should this file be renamed to `single_pods_project_generator_spec.rb`?

Reply:
if renamed, then for the sake of the diff you can also keep the variable to `@generator` too!

Reply:
Good idea